### PR TITLE
properly install systemd unit on multi user target

### PR DESCRIPTION
**Pull request by xh3b4sd**
Fixes https://github.com/giantswarm/aws-operator/issues/131.

### CODE REVIEW

File: templates.go [hunk 1]
```diff
@@ -1384,6 +1384,9 @@ coreos:
 
       [Service]
       ExecStart=/opt/bin/decrypt-tls-assets
+
+      [Install]
+      WantedBy=multi-user.target
```
Comment (xh3b4sd) [inline templates.go]: See https://www.freedesktop.org/software/systemd/man/systemd.unit.html#%5BInstall%5D%20Section%20Options.

**Comment by JosephSalisbury** [comment]
huh

we also have https://github.com/giantswarm/aws-operator/blob/5bb51ace983957697d6ad8afc85cc7b479d040ae/service/create/templates.go

which one of these is source of truth?

**Comment by JosephSalisbury** [comment]
Having both kind of worries me, as two people have worked on two different ones :D

**Comment by xh3b4sd** [comment]
Well, I am pretty lost on this organization of code now. Don't know. We should get this and the concept/idea behind it straight with @nhlfr and @asymmetric.

**Comment by JosephSalisbury** [comment]
see https://github.com/giantswarm/aws-operator/issues/143 for tracking

**Comment by xh3b4sd** [comment]
Having a closer look at the original issue I do not think this will fix it because the install target is defined as you just saw on the CoreOS.

**Comment by nhlfr** [comment]
This service was removed by the another PR, closing.

**Pull request closed without merging**